modules/contacts/contacts.resolver.js

```diff
@@ -0,0 +1,44 @@
+const contactsService = require('./contacts.service');
+
+const contactsQuery = {
+  getContacts: () => contactsService.getContacts(),
+};
+
```

Comment:
For the future. Horondi can have two or more addresses. Consider adding getContactById. It can be one for office, one for the shop and one for the workshop...